source/common/sequencer_impl.cc

```diff
@@ -33,9 +34,7 @@ SequencerImpl::SequencerImpl(
 void SequencerImpl::start() {
   ASSERT(!running_);
   running_ = true;
-  if (start_time_ < time_source_.monotonicTime()) {
```

Comment:
Can you help me understand the history behind the evaluation that was deleted from here and lead to the error message "Sequencer start called too late"?

I am trying to figure out why it is no longer needed and how does it relate to phases.

Reply:
Ah I forgot about this; I'll ping you on Slack, this needs some explaining, and also some polish, as right now we're not doing a no-op. Let me try to share some context here: 

Today, when preparing the worker threads, the main thread computes the times at which load generation should start for each worker. When doing so it tries to carefully offset these timings so that requests that are released by each worker will be evenly spaced in time from a global perspective (we want to avoid that workers all release requests with exactly the same timings). 
This needs some thought now that we introduce phases as well as have the new rate limiters ready for wiring up as features.

Reply:
I made the `phase` execute the wait for the start offset of a worker. So we are doing a no-op again. Thanks for catching this!